cmd/nerdctl/container_prune.go

```diff
@@ -0,0 +1,95 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/cio"
+	"github.com/containerd/containerd/errdefs"
+	"github.com/containerd/containerd/namespaces"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newContainerPruneCommand() *cobra.Command {
+	containerPruneCommand := &cobra.Command{
+		Use:           "prune [flags]",
+		Short:         "Remove all stopped containers",
```

Comment:
Please update `Command Reference` in README.md too

Reply:
And `Unimplemented Docker commands` too

https://github.com/containerd/nerdctl/tree/v0.16.1#unimplemented-docker-commands

Reply:
Both updated!